.github/workflows/maven.yml

```
@@ -34,13 +34,14 @@ jobs:
       TEST_HOME: /home/runner/work/financial-services/financial-services
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)